### PR TITLE
feat(rich-text-editor): fix bug, add font-size and add api demo

### PR DESCRIPTION
**Pull request by Caesar-ch**
# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-vue/blob/dev/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?
feat(rich-text-editor): fix bug, add font-size and add api demo
## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: examples/sites/demos/app/rich-text-editor/event-usage-composition-api.vue [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+}
+const transaction = function (editor) {
+  console.log('transaction', editor);
+}
+const destroy = function (editor) {
+  console.log('destroy', editor);
+}
+const update = function (editor) {
+  console.log('update', editor);
+}
```
Comment (GaoNeng-wWw) [inline examples/sites/demos/app/rich-text-editor/event-usage-composition-api.vue:32]: 这一部分最好使用Notify

File: packages/renderless/src/rich-text-edtior/index.ts [hunk 2]
```diff
@@ -1,13 +1,15 @@
 export const handleChange = (editor) => {
   return (event) => {
     const file = event.target.files[0]
+    console.log(file);
+
```
Comment (GaoNeng-wWw) [inline packages/renderless/src/rich-text-edtior/index.ts]:
```suggestion
```

File: examples/sites/demos/app/rich-text-editor/event-usage-composition-api.vue (hunk 1 above)
Comment (GaoNeng-wWw) [inline examples/sites/demos/app/rich-text-editor/event-usage-composition-api.vue]: 这一段使用Notify组件更好，因为用户不一定会打开控制台

File: examples/sites/demos/app/rich-text-editor/event-usage.vue [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    },
+    transaction(editor) {
+      console.log('transaction', editor);
+    },
+    destroy(editor) {
+      console.log('destroy', editor);
+    },
+    update(editor) {
+      console.log('update', editor);
+    },
```
Comment (GaoNeng-wWw) [inline examples/sites/demos/app/rich-text-editor/event-usage.vue]: 这一段使用Notify组件更好，因为用户不一定会打开控制台

**Review by GaoNeng-wWw — CHANGES_REQUESTED** [review]

File: examples/sites/demos/app/rich-text-editor/event-usage.vue (hunk 3 above)
Comment (Caesar-ch) [reply examples/sites/demos/app/rich-text-editor/event-usage.vue]: 改了

File: examples/sites/demos/app/rich-text-editor/event-usage-composition-api.vue (hunk 1 above)
Comment (Caesar-ch) [reply examples/sites/demos/app/rich-text-editor/event-usage-composition-api.vue]: 改了

**Comment by GaoNeng-wWw** [comment]
有一个冲突需要解决一下 @Caesar-ch

**Review by GaoNeng-wWw — APPROVED** [review]

**Pull request merged**